build/openwisp_nginx/openwisp.ssl.template.conf

```diff
@@ -14,13 +14,22 @@ server {
     ssl_certificate_key  /etc/letsencrypt/live/${DOMAIN}/privkey.pem;
     ssl_session_cache shared:SSL:20m;
     ssl_session_timeout 10m;
-    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_protocols TLSv1.2 TLSv1.3;
     ssl_prefer_server_ciphers on;
-    ssl_ciphers ECDH+AESGCM:ECDH+AES256:ECDH+AES128:DH+3DES:!ADH:!AECDH:!MD5;
+    # generated 2022-02-02, Mozilla Guideline v5.6, nginx 1.17.7, OpenSSL 1.1.1k, intermediate configuration
+    # https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1k&guideline=5.6
+    ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
 
     # Aditional Security Headers
-    add_header Strict-Transport-Security "max-age=31536000";
-    add_header X-Content-Type-Options nosniff;
+    add_header X-XSS-Protection          "1; mode=block" always;
+    add_header X-Content-Type-Options    "nosniff" always;
+    add_header Referrer-Policy           "same-site" always;
+    add_header Content-Security-Policy   "default-src http: https: data: blob: 'unsafe-inline';
+    script-src 'unsafe-eval' https: 'unsafe-inline' 'self';
+    frame-ancestors 'self'; connect-src https://${DOMAIN} wss: 'self';
+    worker-src https://${DOMAIN} blob: 'self';" always;
```

Comment:
Won't we need to make `Content-Security-Policy` configurable like it is in ansible-openwisp2?

Reply:
@codesankalp @atb00ker can we make it consistent with [how it's done in ansible-openwisp2](https://github.com/openwisp/ansible-openwisp2/pull/342/files)?
We could define this as a customizable variable in the docker image for example?